variables.tf

```diff
@@ -1,15 +1,57 @@
-
+variable "bucket_name" {
+  default     = null
+  description = "Name to apply to bucket (use `bucket_name` or `bucket_suffix`)"
+  type        = string
+}
 
 variable "bucket_suffix" {
-  type = string
+  default     = "default"
+  description = "Suffix to apply to the bucket (use `bucket_name` or `bucket_suffix`). When using `bucket_suffix`, the bucket name will be `[ACCOUNT_ID]-[REGION]-s3logging-[BUCKET_SUFFIX]."
+  type        = string
+}
+
+variable "lifecycle_rules" {
+  default = [{
+    id                            = "expire"
+    enabled                       = true
+    prefix                        = null
+    expiration                    = 2147483647
+    noncurrent_version_expiration = 365
+  }]
+
+  description = "lifecycle rules to apply to the bucket"
```

Comment:
Single line values sort above multiline values

Reply:
not the rule

Reply:
Do we not have an actual rule about that? I'll need to open a PR. It is in Terraform's style guide I'm 90% sure